cumulusci/utils/__init__.py

```diff
@@ -417,8 +417,7 @@ def doc_task(task_name, task_config, project_config=None, org_config=None):
     from cumulusci.core.utils import import_global
 
     doc = []
-    doc.append(f"\n\n.. _{task_name}:\n")
-    doc.append(f"**{task_name}**\n==========================================\n")
+    doc.append(f"{task_name}\n==========================================\n")
```

Comment:
```suggestion
    doc.append(f"{task_name}\n{"=" * len(task_name)}\n")
```

just to tidy up the length of the underline so sphinx doesn't complain about it.

Reply:
Also it looks like there's a test adjustment needed